packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -0,0 +1,100 @@
+import json
+import requests
+
+RANGE_ENDPOINT = '/system/v1/logs/v1/range/'
+STREAM_ENDPOINT = '/system/v1/logs/v1/stream/'
+
+
+def validate_json_entry(entry):
+    required_top_level_fields = ['fields', 'cursor', 'monotonic_timestamp', 'realtime_timestamp']
+
+    assert isinstance(entry, dict), 'Entry must be dict'
+    assert entry, 'Entry cannot be empty'
+
+    for required_field in required_top_level_fields:
```

Comment:
why not do a set comparison here? Then we get all the non-matching keys + py.test's assert introspection

`assert set(entry.keys()) <= required_fields, "Entry didn't have all required fields. Entry fields: {}, required fields:{}".format(entry, required_fields)`